src/api/cpp/cvc5_kind.h

```diff
@@ -78,187 +82,227 @@ enum CVC4_EXPORT Kind : int32_t
 #endif
   /**
    * Equality, chainable.
+   * 
    * Parameters: n > 1
-   *   -[1]..[n]: Terms with same sorts
+   *   - 1..n: Terms with same sorts
+   * 
    * Create with:
-   *   mkTerm(Kind kind, Term child1, Term child2)
-   *   mkTerm(Kind kind, Term child1, Term child2, Term child3)
-   *   mkTerm(Kind kind, const std::vector<Term>& children)
+   *   - `mkTerm(Kind kind, Term child1, Term child2)`
+   *   - `mkTerm(Kind kind, Term child1, Term child2, Term child3)`
+   *   - `mkTerm(Kind kind, const std::vector<Term>& children)`
    */
   EQUAL,
   /**
    * Disequality.
+   * 
    * Parameters: n > 1
-   *   -[1]..[n]: Terms with same sorts
+   *   - 1..n: Terms with same sorts
+   * 
    * Create with:
-   *   mkTerm(Kind kind, Term child1, Term child2)
-   *   mkTerm(Kind kind, Term child1, Term child2, Term child3)
-   *   mkTerm(Kind kind, const std::vector<Term>& children)
+   *   - `mkTerm(Kind kind, Term child1, Term child2)`
+   *   - `mkTerm(Kind kind, Term child1, Term child2, Term child3)`
+   *   - `mkTerm(Kind kind, const std::vector<Term>& children)`
    */
   DISTINCT,
   /**
    * First-order constant.
+   * 
    * Not permitted in bindings (forall, exists, ...).
+   * 
    * Parameters:
-   *   See mkConst().
+   *   - See @ref cvc5::api::Solver::mkConst() "mkConst()".
+   * 
    * Create with:
-   *   mkConst(const std::string& symbol, Sort sort)
-   *   mkConst(Sort sort)
+   *   - `mkConst(const std::string& symbol, Sort sort)`
+   *   - `mkConst(Sort sort)`
    */
   CONSTANT,
   /**
    * (Bound) variable.
+   * 
    * Permitted in bindings and in the lambda and quantifier bodies only.
+   * 
    * Parameters:
-   *   See mkVar().
+   *   - See @ref cvc5::api::Solver::mkVar() "mkVar()".
+   * 
    * Create with:
-   *   mkVar(const std::string& symbol, Sort sort)
-   *   mkVar(Sort sort)
+   *   - `cvc5::api::Solver::mkVar(Sort sort, const std::string& symbol)`
```

Comment:
Why do we need prefix `cvc5::api` here but not above, for `mkConst` or `mkTerm`?

Reply:
We don't need it.